configure.ac

```diff
@@ -0,0 +1,200 @@
+AC_PREREQ([2.68])
+AC_INIT([rkt], [0.6.1+git], [https://github.com/coreos/rkt/issues])
```

Comment:
I have an item in TODO about replacing the script with something like `make VER=0.7.0 bump`.